### ISSUE TITLE
Manually-specified variables were not used by the project

### ISSUE THREAD

**Issue by justablob** [open-quantum-safe/liboqs-rust#29]
When building on Windows, this error occurs and the build fails:
```
Chef@WJ68NAB4 ~\App\shared
$ cargo build
    Blocking waiting for file lock on package cache
    Updating git repository `https://github.com/open-quantum-safe/liboqs-rust`
    Updating crates.io index
    Blocking waiting for file lock on package cache
    Blocking waiting for file lock on package cache
    Blocking waiting for file lock on build directory
   Compiling regex-syntax v0.6.22
   Compiling regex v1.4.3
   Compiling env_logger v0.8.3
   Compiling bindgen v0.57.0
   Compiling oqs-sys v0.5.0 (https://github.com/open-quantum-safe/liboqs-rust?branch=main#67abcfcb)
error: failed to run custom build command for `oqs-sys v0.5.0 (https://github.com/open-quantum-safe/liboqs-rust?branch=main#67abcfcb)`

Caused by:
  process didn't exit successfully: `C:\Users\Chef\App\shared\target\debug\build\oqs-sys-318ffd7313b93664\build-script-build` (exit code: 101)
  --- stderr
  CMake Warning:
    Manually-specified variables were not used by the project:

      CMAKE_BUILD_TYPE
      CMAKE_CXX_FLAGS
      CMAKE_CXX_FLAGS_RELEASE


  thread 'main' panicked at '
  command did not execute successfully, got: exit code: 1

  build script failed, must exit now', C:\Users\Chef\.cargo\registry\src\github.com-1ecc6299db9ec823\cmake-0.1.45\src\lib.rs:894:5
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Versions:
- Rust 1.51.0-nightly (d1aed50ab 2021-01-26)
- cmake 3.19.4

**Comment by thomwiggers** [comment]
Coudl you run with `RUST_BACKTRACE=1`?

**Comment by justablob** [comment]
Yep, here you go: https://hatebin.com/zuidowufhq

**Comment by thomwiggers** [comment]
And these `CMAKE_` variables are not set in your own environment?

**Comment by justablob** [comment]
Nope, none of them are set.

**Comment by thomwiggers** [comment]
Weird. Maybe #30 will surface this issue.

**Comment by BlackHoleFox** [comment]
@justablob I have ran into this exact issue trying to build `oqs` on Windows. The error you are seeing probably isn't the full one, at least it wasn't for me. Try running `cargo build 2>&1 | Out-File ./compile.txt`. That should capture everything from the build process, including the C compiler errors. Do the contents of the file end up containing [this string bit](https://github.com/open-quantum-safe/liboqs-rust/issues/17#issuecomment-778073254)?

**Comment by thomwiggers** [comment]
I don't think this is a problem of this crate and just a symptom of the problems solved by #30

**Comment by thomwiggers** [comment]
#30 was merged, would you mind checking if it's fixed?
